components/LayoutSingle.js

```diff
@@ -0,0 +1,16 @@
+import { Footer } from './Footer';
+import { HeaderSingle } from './HeaderSingle';
+
+export const LayoutSingle = ({ children }) => (
+  <>
+    <HeaderSingle />
```

Comment:
Since this component is very small. You can happily put here the HeaderSingle directly.